extensions/wikia/AdEngine/js/video/videoAdFactory.js

```diff
@@ -24,6 +24,7 @@ define('ext.wikia.adEngine.video.videoAdFactory', [
 				this.ima.addEventListener(eventName, callback);
 			},
 			play: function (width, height) {
+				console.log('***** VIDEO AD FACTORY CLICK');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)